db/migrate/20150823120001_namespace_ems_openstack_availability_zones_null.rb

```diff
@@ -0,0 +1,18 @@
+class NamespaceEmsOpenstackAvailabilityZonesNull < ActiveRecord::Migration
+  include MigrationHelper
+
+  NAME_MAP = Hash[*%w(
+    AvailabilityZoneOpenstackNull               ManageIQ::Providers::Openstack::CloudManager::AvailabilityZoneNull
+  )]
+
+  def change
+    # Fix issues where future migrations could be named incorrectly due to the
+    #   bad naming of this particular migration
+    bad = Pathname.glob(Rails.root.join("db/migrate/20151435*")).first
```

Comment:
Yeah, but it's not expensive, and it will prevent accidental commits of bad migration numbers.  At the very least, I'd like to keep it until the bot can be made smarter.